.github/workflows/ci.yml

```diff
@@ -174,8 +160,14 @@ jobs:
         run: |
           V=1 bash scripts/build_p2pd.sh p2pdCache 124530a3
 
-      - name: Run nim-libp2p tests
+      - name: Run tests
         run: |
+          if [[ "${{ matrix.target.os }}" == "windows" ]]; then
+            # https://github.com/status-im/nimbus-eth2/issues/3121
+            export NIMFLAGS="-d:nimRawSetjmp"
+          fi
```

Comment:
Shouldn't this be set in the `config.nims` instead?

Reply:
Maybe, but not all projects have one and it's more important to me to be able to use "--skipParentCfg" than concentrating all compiler flags in a single place.